pegasus/sites.v3/code.org/views/index_video_reveal.haml

```diff
@@ -12,9 +12,9 @@
   - unless download_filename.nil_or_empty?
     %div{style: "float:right"}
       %a{href: "http://s3.amazonaws.com/cdo-videos/#{download_filename}"}
-        %img{width: 34, height: 34, style: "margin-top: 10px;", src: "/images/download_cropped.png"}
+        %img{width: 34, height: 34, style: "margin-top: 10px;", src: "/images/download_cropped.png", alt: "Download video"}
 
   - unless download_path.nil_or_empty?
     %div{style: "float:right"}
       %a{href: download_path}
-        %img{width: 34, height: 34, style: "margin-top: 10px;", src: "/images/download_cropped.png"}
+        %img{width: 34, height: 34, style: "margin-top: 10px;", src: "/images/download_cropped.png", alt: "Download video"}
```

Comment:
better to be accessible in english-only than for no one at all, but I'm wondering if alt text like this is supposed to be translated too?

Reply:
I think ideally it should be, but making it so is probably out of scope for this task since nothing else in these partials appears to be translated. I have a [ticket](https://codedotorg.atlassian.net/browse/ACQ-583?atlOrigin=eyJpIjoiZTlhY2RmNDhhMGE0NDlmMGFjOWRiNDdkNTM2OGMxZTMiLCJwIjoiaiJ9) to reevaluate if this is the right pattern to use for videos in general and will make a note there!

Reply:
haha, yes, if the page is not translated then this is definitely enough for today 👍